src/main/java/forestry/apiculture/BeeHousingModifier.java

```diff
@@ -49,8 +49,7 @@ public float getLifespanModifier(IBeeGenome genome, IBeeGenome mate, final float
 
     @Override
     public float getProductionModifier(IBeeGenome genome, final float currentModifier) {
-        float modifierValue = 0.0f;// the "base" value of 1.0 should come from Bee.java or BeeKeepingMode.jave, that is,
-                                   // it should be set at the call site
+        float modifierValue = 1.0f;
```

Comment:
That was actually right, the new production modifier is supposed to start at 0, therefore the normal apiary without frames should have -0.9 (not 0.1) by default and for magic apiary it should be -0.1 (not 0.9). Please revert this commit.

Reply:
Meant to just click confirm but pushing changes manually removed that option.

Fixed in https://github.com/GTNewHorizons/ForestryMC/pull/55/commits/96afda431fe7519fb2cd0e31dbefbcdd0335b196